run.sh

```diff
@@ -39,36 +41,40 @@ main() {
     "$jq" -r "." "$WERCKER_STEP_TEMP/result.json"
     fail "Failed to start Runscope bucket tests."
   else
-    BUCKET_KEY=$(extract_json_value ".data.runs | .[0] | .bucket_key" "$WERCKER_STEP_TEMP/result.json")
-    TEST_ID=$(extract_json_value ".data.runs | .[0] | .test_id" "$WERCKER_STEP_TEMP/result.json")
-    TEST_RUN_ID=$(extract_json_value ".data.runs | .[0] | .test_run_id" "$WERCKER_STEP_TEMP/result.json")
-    RUNSCOPE_TEST_MONTIOR_URL="https://api.runscope.com/buckets/$BUCKET_KEY/tests/$TEST_ID/results/$TEST_RUN_ID"
+    if [ "$WERCKER_RUNSCOPE_BUCKET_TRIGGER_SKIP_MONITORING" = "true" ]; then
+      info "Skip waiting for test to complete. Please go directly to Runscope to see the results."
```

Comment:
Ah yes, the response of this trigger has the URL. I'll extract it and display it as an `info message`.